.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
```

Comment:
If we only use Github for bug reports, is it useful adding this to the title and using the label?

At most I think just use the label (in case we change our use of Github in the future), and leave it out of the name (which seems duplicative).